### ISSUE TITLE
Huge queue on Windows runner

### ISSUE THREAD

**Issue by huydhn** [pytorch/pytorch#129383]
https://hud.pytorch.org/metrics

## Current Status
Ongoing

## Error looks like
*Provide some way users can tell that this SEV is causing their issue.*

## Incident timeline (all times pacific)
Since Friday, June 21st

## User impact
User cannot merge normally

## Root cause
TBD

## Mitigation
Going to disable some CI jobs:
* https://github.com/pytorch/pytorch/issues/129388

**Comment by seemethere** [comment]
Looks like windows runners are failing to even start up:

```
C:\Windows\system32\config\systemprofile\AppData\Local\Temp\Amazon\EC2-Windows\Launch\InvokeUserData\UserScript.ps1:89
char:17
+               --password-stdin "$AWS_ACCOUNT_ID.dkr.ecr.$AWS_DEFAULT_ ...
+                 ~
Missing expression after unary operator '--'.
At
C:\Windows\system32\config\systemprofile\AppData\Local\Temp\Amazon\EC2-Windows\Launch\InvokeUserData\UserScript.ps1:89
char:17
+               --password-stdin "$AWS_ACCOUNT_ID.dkr.ecr.$AWS_DEFAULT_ ...
+                 ~~~~~~~~~~~~~~
Unexpected token 'password-stdin' in expression or statement.
    + CategoryInfo          : ParserError: (:) [], ParseException
    + FullyQualifiedErrorId : MissingExpressionAfterOperator


2024/06/24 18:27:08Z: Message: The output from user data script: Refreshing environment variables from the registry for powershell.exe. Please wait...
Finished

2024/06/24 18:27:08Z: User data script completed.
```

From our current userdata logs output.

Found by ssh'ing into a new AWS instance and doing `cat C:\ProgramData\Amazon\EC2-Windows\Launch\Log\UserdataExecution.log`

**Comment by seemethere** [comment]
Re-opening since the fix hasn't been rolled out yet.

TL;DR of what happened:

1. A PR was made in the private repository to change our post install script for how we log into ECR for every instance (https://github.com/pytorch-labs/pytorch-gha-infra/pull/416)
2. A subsequent PR was made to resolve issues that surface when that PR was landed: https://github.com/pytorch/test-infra/pull/5318
3. Through a series of unfortunate lands + reverts of the AL2023 migration we ended up in a state where the initial PR was deployed but the subsequent PR fix was not deployed, which caused a state where Windows runners could not get past the initial setup phase.

**Comment by huydhn** [comment]
In term of timeline, 2 was landed before 1 from what I see, so there is a bug somewhere either in https://github.com/pytorch/test-infra/pull/5318 or in the deployment process I think.  We ran the deployment 4 days ago https://github.com/pytorch-labs/pytorch-gha-infra/actions/runs/9600196913, so both should be there theoretically

**Comment by huydhn** [comment]
https://github.com/pytorch-labs/pytorch-gha-infra/pull/419 helps, I spot check a newly launch runner and it picks up jobs now https://github.com/pytorch/pytorch/actions/runs/9629882001/job/26559829793

**Comment by seemethere** [comment]
Alright! I see we have runners now, going to go ahead and close this one!
